src/dlgTriggerEditor.cpp

```diff
@@ -125,68 +119,56 @@ dlgTriggerEditor::dlgTriggerEditor( Host * pH )
     setStatusBar( statusBar );
     statusBar->show();
     mIsGrabKey = false;
-    QVBoxLayout * pVB1 = new QVBoxLayout(mainArea);
+
+    // Replacement code to use single splitter for all right widgets:
+    mpSystemMessageArea = new dlgSystemMessageArea(splitter_rightTopBottom);
```

Comment:
Well spotted - I wasn't aware of that one and it was introduced in Qt 5.0 so I only need a replacement for my Frankensteinian(?) Qt 4.x backport {_As Gene Wilder in 'Young Frankenstein':_ "My God! It is alive!..."}